src/utils/Constants.ts

```diff
@@ -46,32 +46,74 @@ export const FORMATS = {
   rpm: 'rpm'
 };
 
+export const PURL_TYPES = {
+  alpine: 'alpine',
+  cargo: 'cargo', //cargo == crates == rust
+  chocolatey: 'chocolatey',
+  clojars: 'clojars',
+  cocoapods: 'cocoapods',
+  composer: 'composer', //packagist website but composer format, php language
+  conan: 'conan',
+  conda: 'conda',
+  cran: 'cran',
+  debian: 'deb',
+  gem: 'gem',
+  github: 'github',
+  golang: 'golang',
+  maven: 'maven',
+  npm: 'npm',
+  nuget: 'nuget',
+  pypi: 'pypi',
+  rpm: 'rpm'
+};
```

Comment:
What is this for?

Reply:
We need to make the FORMATS the same as the PURL_TYPES. So we should drop PURL_TYPES. I will delete it.